client/src/components/ProductList/ProductList.js

```diff
@@ -4,7 +4,30 @@ import styled from 'styled-components/macro'
 import { NavLink } from 'react-router-dom'
 
 export default function ProductList(props) {
-  const { styleId, media, brand, model, retailPrice, sale } = props.product
+  const { styleId, media, brand, model, shops, sale } = props.product
+
+  function shopCount(count) {
+    if (count > 1) {
+      return count + ' Shops'
+    } else {
+      return count + ' Shop'
+    }
+  }
```

Comment:
Just another way to write it :)
```suggestion
  const shopCount = count => count > 1 ? count + ' Shops' : count + ' Shop'
```